Classes/Client.php

```diff
@@ -27,6 +28,15 @@ public function run()
             HttpUtility::setResponseCodeAndExit(HttpUtility::HTTP_STATUS_403);
         }
 
+        EidUtility::initTCA();
```

Comment:
i will do .. it was necessary to have a full loaded tca for backend users extbase usage